ecosystem/sep-0024.md

```diff
@@ -107,6 +109,78 @@ non-equivalent asset transactions, specifically `amount_in_asset`, `amount_fee_a
 allows client applications can relay this information, in addition to the amounts specified in the associated
 attributes, back to their users.
 
+Alternatively, the Anchor can support the [SEP-38] RFQ API and the `quote_id` attribute in
+`POST /transactions/deposit/interactive` and `POST /transactions/withdraw/interactive` requests. This enables wallets to
+check the Anchor's exchange rates prior to initiating a transaction and obtain a firm quote to use for an upcoming
+transaction.
+
+### Asset Conversions Using the Market Rate
+
+To convert at the market rate, clients get an
+[indicative quote](https://www.investopedia.com/terms/i/indicativequote.asp) with [SEP-38] and then proceed to calling
+the [`POST /transactions/withdraw/interactive`](#withdraw) or [`POST /transactions/deposit/interactive`](#deposit)
+endpoints. Market rate conversions executed in this way do not provide a guarantee of the final price to be paid for the
+asset. Here is how this kind of conversion should happen in deposits and withdrawals for an asset pair like
+`BRL <> USDC`:
+
+Anchors offer indicative quote support via enabling `indicative_quotes_supported` feature flag.
+
+#### Deposit:
+
+Considering the user will be making a bank deposit of fiat `BRL` in exchange for `USDC` in the Stellar network:
+
+1. The Wallet checks the assets' market conversion rate with [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the
+   estimated price to the user.
+1. The Wallet initiates a [`POST /transactions/deposit/interactive`](#deposit) using the parameter `asset_code`.
+1. The user proceeds with the off-chain deposit, sending `BRL` to the Anchor's bank account.
+1. Upon receiving the off-chain deposit funds, the Anchor converts it to `USDC` using the market rate and sends the
+   resulting amount to the user's Stellar account.
+
+#### Withdraw:
+
+Considering the user will be making a Stellar payment of `USDC` in exchange for withdrawing `BRL` in the bank account:
+
+1. The Wallet checks the assets' market conversion rate with [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the
+   estimated price to the user.
+1. The Wallet initiates a [`POST /transactions/withdraw/interactive`](#withdraw) using the parameter `asset_code`.
+1. The user proceeds with the on-chain payment, sending `USDC` to the Anchor's Stellar account.
+1. Upon receiving the on-chain withdraw funds, the Anchor converts it to `BRL` using the market rate and sends the
+   resulting amount to the user's bank account.
+
+### Asset Conversions Using a Firm Quote
+
+With firm quotes, users are guaranteed an asset price they will pay if they submit the payment and anchor receives the
+payment before the quote expiration. Here is how the conversion should happen in deposits and withdrawals for an asset
+pair like `USDC <> BRL`:
+
+Anchors enable firm quotes support with `firm_quotes_supported` feature flag.
```

Comment:
If we remove `indicative_quotes_supported`, we can update this to `quotes_supported`.